### ISSUE TITLE
`alacritty(5)` and `/config-alacritty.html` does not actually document where to place the config file

### ISSUE THREAD

**Issue by strugee** [alacritty/alacritty#8098]
### System

N/A, but I happen to be on macOS

### Logs

N/A

### STR

1. Visit https://alacritty.org/
2. Click "Configuration" in the header bar
3. Try and figure out where to place Alacritty's config file, or what it should even be called

I would expect this to either be in the SYNOPSIS (see [`veritytab(5)`](https://manpages.ubuntu.com/manpages/oracular/en/man5/veritytab.5.html) as a random example), or immediately described in the DESCRIPTION (see for example [`ssh_config(5)`](https://manpages.ubuntu.com/manpages/oracular/en/man5/ssh_config.5.html)).

**Comment by kchibisov** [comment]
it's in `alacritty(1)`, could probably duplicate in `alacritty(5)` as well...

**Comment by strugee** [comment]
Yeah, a coworker just pointed me at `alacritty(1)` too. Duplication would be nice.

My one concern is that if it's documented in >1 place, then the documentation might get out of sync. I don't really buy that since this seems like it should change basically never, but if that's a concern, maybe just point to `alacritty(1)` in `alacritty(5)`.

FWIW, this was particularly brutal for me because I just downloaded the macOS app bundle on alacritty.org. That meant that `man alacritty` and friends did not work out of the box, so unless I wanted to do e.g. `man /Applications/Alacritty.app/Contents/Resources/alacritty.1.gz` (and knew to even look there), the only way of reading the manpages was via the web... but `alacritty(1)` in the SEE ALSO section isn't a link. So I was pretty much SOL.

**Comment by strugee** [comment]
(Happy to PR if duplication is fine, though I suspect there's still a few other things that could have gone better here)

**Comment by chrisduerr** [comment]
I'm fine with putting a little section at the top that tells people where the file is supposed to go.

**Comment by chrisduerr** [comment]
> but alacritty(1) in the SEE ALSO section isn't a link.

If you believe this to be an issue, feel free to open a separate issue or throw up a PR on the website repository tbh. Generating the docs for all the manpages even if they're just accessible through the config root shouldn't be an issue.

**Comment by nixpulvis** [comment]
Isn't this addressed by 26ed8e1b9dfe2e4c5d23dc80043554197113d83f?
